server/service/src/processors/transfer/shipment/create_inbound_shipment.rs

```diff
@@ -119,6 +119,24 @@ fn generate_inbound_shipment(
         .as_ref()
         .map(|r| r.requisition_row.id.clone());
 
+    let reference_string = [
+        "Reference",
```

Comment:
I think `format!("Reference: {} {}", ..)` is easier to read, can also do: 
`outbound_shipment_row.comment.as_ref().unwrap_or_default()`

Reply:
[a17c9f7](https://github.com/openmsupply/open-msupply/pull/664/commits/a17c9f70ce52a9310503369bffc2dbd26ac9e91a)

Reply:
can you also use format for requisition please

Reply:
It is? 🤔